cuml_bench/dbscan.py

```diff
@@ -0,0 +1,54 @@
+#===============================================================================
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import sys
+import os
+import argparse
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))

```

Comment:
Does more simple method to add bench in python view exist?
May be, shorter `sys.path.append(os.path.dirname(os.getcwd()))`?

Reply:
I tried. Not works.